src/sidebar/components/MarkdownEditor.tsx

```diff
@@ -146,6 +148,11 @@ function handleToolbarCommand(
     case 'list':
       update(state => toggleBlockStyle(state, '* '));
       break;
+    case 'mention':
+      update(state => toggleSpanStyle(state, '@', '', ''));
+      // Dispatch keyup event with `@` key, to open the suggestions popover
+      inputEl.dispatchEvent(new KeyboardEvent('keyup', { key: '@' }));
```

Comment:
This couples a bit the logic here with how the suggestions popover is displayed. I tried to make this work transparently but failed miserably, and any other approach I came up with required a bunch of extra changes anyway.

This seems simple enough, but I'm open to suggestions.

Reply:
I think this is fine.